<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QWidget" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="studenPages">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="stuMainPage">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pBStuInfo">
       <property name="text">
        <string>个人信息</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pBStuCho">
       <property name="text">
        <string>学生选课</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pBStuPlan">
       <property name="text">
        <string>培养计划</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pBStuExam">
       <property name="text">
        <string>成绩信息</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pBStuTimetable">
       <property name="text">
        <string>课表查询</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pBStuExam_2">
       <property name="text">
        <string>考试信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="stuInfo"/>
   <widget class="QWidget" name="stuChooselesson"/>
   <widget class="QWidget" name="stuPlan"/>
   <widget class="QWidget" name="stuGrade"/>
   <widget class="QWidget" name="stuTimetable"/>
   <widget class="QWidget" name="stuExams"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
